codegen/proto/options.go

```diff
@@ -256,3 +256,13 @@ func getFieldOptionOpenAPIValidationDisabled(field *descriptor.FieldDescriptorPr
 	}
 	return false, nil
 }
+
+func lowerCamelName(field FieldOptions) string {
+	name := field.Field.GetName()
+	// Hack because strcase.ToLowerCamel is not converting string the same as the `cue` lib is
+	if name == "k8s" {
```

Comment:
How confident are we that this string is always already lowercase? Should we do a name.ToLower() here?

Reply:
This is right I confirmed this change fixes the generated CRDs locally